recipe/meta.yaml

```diff
@@ -29,6 +29,9 @@ outputs:
       license: BSD-3-Clause
       license_family: BSD
       summary: A meta-package to select CPU or GPU UCX build.
+    extra:
+      variants:
+        cuda_compiler_version: {{ cuda_compiler_version }}
```

Comment:
Had to add this hack as conda-build complained (see exception below and [this log]( https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=73992 )) that the `cuda_compiler_version` variant was not used. `extra` is a hacky workaround for this problem that has really no other impact on the package except to ensure this variant is included.

<details>
<summary>Exception:</summary>

```python
TEST START: /home/conda/feedstock_root/build_artifacts/linux-64/ucx-proc-1.0.0-cpu.tar.bz2
Traceback (most recent call last):
  File "/opt/conda/lib/python3.7/site-packages/conda_build/variants.py", line 514, in get_package_combined_spec
    validate_spec(spec)
  File "/opt/conda/lib/python3.7/site-packages/conda_build/variants.py", line 138, in validate_spec
    raise ValueError("Variant configuration errors: \n{}".format(errors))
ValueError: Variant configuration errors: 
["zip_key entry cuda_compiler_version in group ['cuda_compiler_version', 'docker_image'] does not have any settings"]

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/opt/conda/bin/conda-build", line 11, in <module>
    sys.exit(main())
  File "/opt/conda/lib/python3.7/site-packages/conda_build/cli/main_build.py", line 445, in main
    execute(sys.argv[1:])
  File "/opt/conda/lib/python3.7/site-packages/conda_build/cli/main_build.py", line 436, in execute
    verify=args.verify, variants=args.variants)
  File "/opt/conda/lib/python3.7/site-packages/conda_build/api.py", line 209, in build
    notest=notest, need_source_download=need_source_download, variants=variants)
  File "/opt/conda/lib/python3.7/site-packages/conda_build/build.py", line 2349, in build_tree
    test(pkg, config=metadata.config.copy(), stats=stats)
  File "/opt/conda/lib/python3.7/site-packages/conda_build/build.py", line 2044, in test
    config)
  File "/opt/conda/lib/python3.7/site-packages/conda_build/build.py", line 1855, in construct_metadata_for_test
    m, hash_input = _construct_metadata_for_test_from_package(recipedir_or_package, config)
  File "/opt/conda/lib/python3.7/site-packages/conda_build/build.py", line 1791, in _construct_metadata_for_test_from_package
    reset_build_id=False)[0][0]
  File "/opt/conda/lib/python3.7/site-packages/conda_build/render.py", line 808, in render_recipe
    m.config.variants = get_package_variants(m, variants=variants)
  File "/opt/conda/lib/python3.7/site-packages/conda_build/variants.py", line 547, in get_package_variants
    combined_spec, specs = get_package_combined_spec(recipedir_or_metadata, config=config, variants=variants)
  File "/opt/conda/lib/python3.7/site-packages/conda_build/variants.py", line 516, in get_package_combined_spec
    raise ValueError("Error in config {}: {}".format(f, str(e)))
ValueError: Error in config /tmp/tmpkblhazkj/info/recipe/conda_build_config.yaml: Variant configuration errors: 
["zip_key entry cuda_compiler_version in group ['cuda_compiler_version', 'docker_image'] does not have any settings"]
```

</details>

Reply:
Try changing `""` to `"cpu"`

Reply:
I originally had `"None"` unfortunately that causes a different set of problems earlier on. Please see [this comment]( https://github.com/conda-forge/ucx-split-feedstock/pull/14#discussion_r326662667 ) for details.

Reply:
I don't think your skipping condition was correct then. `# [cuda_compiler_version != None]`

Reply:
Good point. Will take another look and see if we can't fix that.

Reply:
Have fixed it to work with `"None"` and got rid of this workaround. Thanks for pointing that out 🙂